### PR TITLE
Fix form focus directive, add tests

### PR DESCRIPTION
**Pull request by marcysutton**
When writing a test for #27, I found that the code could stand to be refactored. By adding `scope: true`, support for multiple forms is handled correctly without having to pass around the form name. I also eliminated the `for...in` loop since it was unnecessary. My only remaining question is whether this adequately handles all error cases.

cc @iandotkelly @dylanb

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ module.exports = function(grunt) {
                 banner: ['/*!',
                     '* Angular Directives For Accessible Applications',
                     '*',
-                    '* Copyright (C) 2015 Deque Systems Inc., All Rights Reserved',
+                    '* Copyright (C) 2017 Deque Systems Inc., All Rights Reserved',
```
Comment (dylanb) [inline Gruntfile.js]: should be 2015 - 2017

File: dist/nga11yfocus.min.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /*!
 * Angular Directives For Accessible Applications
 *
-* Copyright (C) 2015 Deque Systems Inc., All Rights Reserved
+* Copyright (C) 2017 Deque Systems Inc., All Rights Reserved
```
Comment (dylanb) [inline dist/nga11yfocus.min.js]: same here

**Review by dylanb — CHANGES_REQUESTED** [review]

File: Gruntfile.js (hunk 1 above)
Comment (marcysutton) [inline Gruntfile.js]: Done

**Comment by chandramuralis** [comment]
Looks fine to me. My only concern is now we have a separate scope, if that's ok with the design I'm fine with it.

**Review by dylanb — APPROVED** [review]

**Comment by chandramuralis** [comment]
Is this released yet?

**Comment by marcysutton** [comment]
It is now! I was able to preserve your commits in this PR, so they are recognized in the code history. Thanks for your contributions! https://www.npmjs.com/package/ng-a11y

**Pull request merged**